src/sql/workbench/parts/notebook/browser/cellViews/code.component.ts

```diff
@@ -305,7 +305,7 @@ export class CodeComponent extends AngularDisposable implements OnInit, OnChange
 	}
 
 	private setFocusAndScroll(): void {
-		if (this.cellModel.id === this._activeCellId) {
+		if (this.cellModel.id === this._activeCellId && this._editor.getContainer().offsetParent) {
```

Comment:
So for those of us who don't know / had to online search, this indicates that the element isn't visible since in cases where it's display is none, this returns null? Perhaps a comment / encapsulate in a method so that it's clearer and folks can find how to use this later?